runtime/str.go

```diff
@@ -201,20 +201,38 @@ func strGE(f *Frame, v, w *Object) (*Object, *BaseException) {
 	return strCompare(v, w, False, True, True), nil
 }
 
+// strGetItem returns a slice of string
```

Comment:
@trotterdylan, fix that, and also added unicode slicing, should I use separate PR?